prov/gni/src/gnix_vc.c

```diff
@@ -1533,10 +1538,10 @@ static int __gnix_vc_rx_progress(struct gnix_vc *vc)
 
 	ret = __gnix_vc_connected(vc);
 	if (ret) {
-		/* The CM will schedule the VC when the connection is complete.
-		 * Return success to allow continued VC RX processing. */
+		/* The CM will schedule the VC when the connection is
```

Comment:
Do we know if this comment is correct?  Is the CM scheduling a VC after the connection completes?